docs/user-guide/cli-using-using-team-profiles.md

```diff
@@ -39,7 +65,7 @@ Consider the following benefits of using team profiles:
 
 ## Important information about team profiles
 
-With the introduction of team profiles, the Secure Credential Store (SCS) Plug-in is deprecated. Secure credential encryption is now handled by the the secure array in the `zowe.config.json` file.
+With the introduction of team profiles, the secure credential store (SCS) Plug-in is deprecated. Secure credential encryption is now handled by the the secure array in the `zowe.config.json` file.
```

Comment:
I think Secure Credential Store should remain capitalized here since it is the proper name for the plug-in.